backend/open_webui/config.py

```diff
@@ -810,6 +810,12 @@ def create_config_file(file_path):
     os.environ.get("ENABLE_MESSAGE_RATING", "True").lower() == "true",
 )
 
+ENABLE_WEBSOCKET_SUPPORT = PersistentConfig(
```

Comment:
Perhaps its better to have this in our new `env.py` file solely as an env var instead of making it persistent?